packages/api/lambdas/create-reconciliation-report.js

```diff
@@ -182,97 +183,115 @@ async function createReconciliationReportForBucket(Bucket, recReportParams) {
   const oneWayBucketReport = isOneWayBucketReport(recReportParams);
   if (oneWayBucketReport) log.debug('Creating one way report, reconciliation report will not report objects only in S3');
 
-  const query = getFilesAndGranuleInfoQuery({
-    knex: recReportParams.knex,
-    searchParams: { bucket: Bucket },
-    sortColumns: ['key'],
-    granuleColumns: ['granule_id'],
-    collectionIds: recReportParams.collectionIds,
-    providers: recReportParams.providers,
-    granuleIds: recReportParams.granuleIds,
-  });
+  return await pRetry(
+    async () => {
+      const query = getFilesAndGranuleInfoQuery({
+        knex: recReportParams.knex,
+        searchParams: { bucket: Bucket },
+        sortColumns: ['key'],
+        granuleColumns: ['granule_id'],
+        collectionIds: recReportParams.collectionIds,
+        providers: recReportParams.providers,
+        granuleIds: recReportParams.granuleIds,
+      });
 
-  const pgFileSearchClient = new QuerySearchClient(query, 100);
+      const pgFileSearchClient = new QuerySearchClient(query, 100);
```

Comment:
Is it not possible to retry the operation under `QuerySearchClient`?

Reply:
https://github.com/nasa/cumulus/pull/3332/commits/c211dac7a088f6dff4d33297170135234210edf5

https://github.com/nasa/cumulus/pull/3332/commits/9308412966a1da9b21b4b68b62aef3789d806bf2